app/js/lib/foxbox/services/base.js

```diff
@@ -232,7 +237,7 @@ export default class BaseService extends EventDispatcher {
    * super.teardown() method as well.
    */
   teardown() {
-    for (let watcher of this[p.watchers].values()) {
+    for (const watcher of this[p.watchers].values()) {
```

Comment:
Right, Fx doesn't support it yet, I've made it const because of transpiling andeslint that was complaining. So if we turn off transpiling later and Fx still doesn't support it we'll have to change it back :/